Kucoin.Net/Interfaces/IKucoinClientSpot.cs

```diff
@@ -640,5 +640,56 @@ public interface IKucoinClientSpot : IRestClient
         /// <param name="ct">Cancellation token</param>
         /// <returns></returns>
         Task<WebCallResult<IEnumerable<KucoinOrder>>> GetStopOrderByClientOrderIdAsync(string clientOrderId, CancellationToken ct = default);
+
+
+        #region Margin Trade
+
+        #region Borrow & Lend
+
+        /// <summary>
+        /// Places a Borrow order (https://docs.kucoin.com/#post-borrow-order)
+        /// </summary>
+        /// <param name="currency">Currency to Borrow e.g USDT etc</param>
+        /// <param name="type">The type of the order (FOK, IOC)</param>
+        /// <param name="quantity">Total size</param>
+        /// <param name="maxRate">The max interest rate. All interest rates are acceptable if this field is left empty</param>
+        /// <param name="term">erm (Unit: Day). All terms are acceptable if this field is left empty. Please note to separate the terms via comma. For example, 7,14,28</param>
+        /// <param name="ct">Cancellation token</param>
+        /// <returns>The id of the new order</returns>
+        Task<WebCallResult<KucoinNewBorrowOrder>> PlaceBorrowOrderAsync(
+           string currency,
```

Comment:
To keep this in line with the rest of the implementations, this should be called `asset`